crates/meroctl/src/cli/config.rs

```diff
@@ -0,0 +1,216 @@
+                    println!("{}", toml::to_string_pretty(&config)?);
+                    return Ok(());
+                }
+                if !self.force {
+                    eyre::bail!(
+                        "The config for the node is already initialized \nYou can override a setting by adding the -f flag"
+                    );
+                } else {
+                    identity = config.identity;
+                    if !boot_network_provided {
+                        self.boot_nodes = config.network.bootstrap.nodes.list;
+                    }
+                    if !swarm_host_provided && !swarm_port_provided {
+                        swarm_listen = Some(config.network.swarm.listen);
+                    }
+                    if !server_host_provided && !server_port_provided {
+                        server_listen = Some(config.network.server.listen);
+                    }
+                    if !mdns_provided {
+                        self.mdns = config.network.discovery.mdns;
+                    }
+                }
+            } else {
+                eyre::bail!(
+                    "Failed to load the Config.toml file for node {}",
+                    root_args.node_name
+                );
+            }
+        } else {
+            eyre::bail!("You have to initialize the node first \nRun command node init -n <NAME>");
+        }
+
+        let mdns = self.mdns && !self.no_mdns;
+
+        let mut listen: Vec<Multiaddr> = vec![];
+
+        match swarm_listen {
+            Some(data) => listen.extend(data),
+            None => {
+                for host in self.swarm_host {
+                    let host = format!(
+                        "/{}/{}",
+                        match host {
+                            std::net::IpAddr::V4(_) => "ip4",
+                            std::net::IpAddr::V6(_) => "ip6",
+                        },
+                        host,
+                    );
+                    listen.push(format!("{}/tcp/{}", host, self.swarm_port).parse()?);
+                    listen.push(format!("{}/udp/{}/quic-v1", host, self.swarm_port).parse()?);
+                }
+            }
+        }
+
+        let mut boot_nodes = vec![];
+        if boot_network_provided {
+            if let Some(network) = self.boot_network {
+                match network {
+                    BootstrapNetwork::CalimeroDev => {
+                        boot_nodes.extend(BootstrapNodes::calimero_dev().list)
+                    }
+                    BootstrapNetwork::Ipfs => boot_nodes.extend(BootstrapNodes::ipfs().list),
+                }
+            }
+        } else {
+            boot_nodes = self.boot_nodes;
+        }
+
+        let config_new = ConfigFile {
+            identity,
+            store: StoreConfig {
+                path: "data".into(),
+            },
+            application: ApplicationConfig {
+                path: "apps".into(),
+            },
```

Comment:
this will override the file contents, we shouldn't be creating a new config.. we should be mutating the existing one

Reply:
I agree, but right now it isn't even an option from the cli to change those two settings.
Should I add an option to change those paths?
Or just leave it but in case the user manually changes his config.toml file?

Reply:
no, we don't need to do that.. just mutate the same object, so you won't have to bother about these